pkg/gen/gogen.go

```diff
@@ -37,19 +38,118 @@ func (gen *CodeGenerator) write(format string, a ...any) {
 	}
 }
 
+// getRequiredImports returns a map of import paths that need to be included
+// based on the field types used in the messages
+func (gen *CodeGenerator) getRequiredImports(messages map[string]spec.Message) map[string]bool {
+	imports := make(map[string]bool)
+
+	// Always needed imports
+	imports["context"] = true
+	imports["fmt"] = true
+	imports["github.com/ostafen/suricata/runtime"] = true
+	imports["github.com/xeipuuv/gojsonschema"] = true
+
+	// Check for type-specific imports
```

Comment:
Maybe it's good idea to put type-specific imports to a separate map, and then add the ones that we need based on actual types found on messages